tests/conftest.py

```diff
@@ -34,6 +34,7 @@ def pytest_runtest_setup(item):
     if test_initialize_marker:
         rng_seed = test_initialize_marker.kwargs["rng_seed"]
         pyro.set_rng_seed(rng_seed)
+    pyro.settings.set(module_local_params=False)
```

Comment:
I believe if we are careful to use context managers in tests, then the default settings should always be active.

Reply:
Done